bob/bob.py

```diff
@@ -1,2 +1,10 @@
 def hey(phrase):
-    pass
+    if phrase.isspace() or not phrase:
+        return 'Fine. Be that way!'
+    if phrase.strip().find('?')+1 == len(phrase.strip()) and not phrase.isupper():
```

Comment:
@zarniak I understand that this is checking if the last character is "?"? In that case, it's good to remember that strings in Python behave much like lists. So, to access the last character of a string, you could do `phrase[-1]`.